examples/hello-world/package.json

```diff
@@ -4,19 +4,20 @@
   "description": "React Intl Hello World Example",
   "private": true,
   "main": "index.js",
-  "scripts": {
-    "clean": "../../node_modules/.bin/rimraf build",
-    "build": "../../node_modules/.bin/mkdirp build && ../../node_modules/.bin/browserify index.js -d -t babelify -t browserify-shim > build/bundle.js",
-    "watch": "../../node_modules/.bin/mkdirp build && ../../node_modules/.bin/watchify index.js -d -t babelify -t browserify-shim -o build/bundle.js -v",
-    "start": "../../node_modules/.bin/babel-node server.js"
-  },
   "author": "Eric Ferraiuolo <edf@ericf.me>",
   "license": "BSD-3-Clause",
-  "browser": {
-    "react-intl": "../../lib/index.js"
+  "devDependencies": {
+    "react-scripts": "0.6.0"
   },
-  "browserify-shim": {
-    "react": "global:React",
-    "react-dom": "global:ReactDOM"
+  "dependencies": {
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2",
+    "react-intl": "^2.1.5"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
```

Comment:
We can remove the `test` command here because it is irrelevant to this example.
Perhaps `eject` too.